Dockerfile

```diff
@@ -150,6 +151,50 @@ RUN mkdir -p /vboxguest && \
     mkdir -p $ROOTFS/sbin && \
     cp x86/lib/VBoxGuestAdditions/mount.vboxsf $ROOTFS/sbin/
 
+# Build VMware Tools
+ENV OVT_VERSION 9.4.6-1770165
+
+# Download and prepare ovt source
+RUN mkdir -p /vmtoolsd && cd /vmtoolsd && \
+    curl -L -o open-vm-tools-$OVT_VERSION.tar.gz http://downloads.sourceforge.net/open-vm-tools/open-vm-tools-$OVT_VERSION.tar.gz && \
+    tar xfz open-vm-tools-$OVT_VERSION.tar.gz && rm open-vm-tools-$OVT_VERSION.tar.gz &&\
+    mv open-vm-tools-$OVT_VERSION open-vm-tools
```

Comment:
Makes sense, it's that I'm always a bit anxious when piping stuff out of curl :smile: let me fix that